app/forms/journeys/further_education_payments/select_provision_form.rb

```diff
@@ -0,0 +1,42 @@
+module Journeys
+  module FurtherEducationPayments
+    class SelectProvisionForm < Form
+      attribute :school_id, :string # school GUID
+
+      validates :school_id, presence: {message: i18n_error_message(:blank)}
+
+      def radio_options
+        results.map do |school|
```

Comment:
How come we need to map these to an open struct? Can we not just return the schools given they have the same api?

Reply:
ah yes I see your point. let me remove the `map` and `openstruct` as they're not necessary